app/move/controllers/create/save.js

```diff
@@ -25,34 +23,6 @@ class SaveController extends CreateBaseController {
       next(error)
     }
   }
-
-  successHandler(req, res) {
```

Comment:
My only concern with removing this altogether is it brings in the case where a user could navigate back with the browser buttons and see the form fully populated and can submit again, creating a duplicate record.

It also brings in the possibility someone having bookmarked the second step and seeing all the details filled in because it wasn't cleared after saving that move to the database.

We probably want a way to separate these two pieces of information. Maybe we create another model for confirmation or make use of the default `req.session` object to store the information we need for the confirmation page?

Trying to think how we've done this in the past with a confirmation page actually 🤔

Reply:
This has been added back Re: the discussion around https://design-system.service.gov.uk/patterns/confirmation-pages/#help-users-who-bookmark-the-page

Have a look over and we can catch up tomorrow